lesson10/jquery.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const bodyParser= require("body-parser");
+const app = express();
+const gm = require("gm");
+const fs = require("fs");
+
+app.listen(3615);
+app.use(express.static('public'));
+app.use(bodyParser.json({limit:'500mb'}));
+
+app.post('/nameCom', (req,res)=>{
+  fs.writeFile('/home/sya/garageScript/lesson10/public/ncom.txt', ` Name: ${req.body.name} Comment: ${req.body.comment}`)
+  res.send("name and comment have been saved ");
+})
+
+
+app.post('/pics', (req,res)=>{
+  let picData = req.body.canvas.replace('data:image/png;base64',' ');
+  let picPath = `/home/sya/garageScript/lesson10/public/pics/${req.body.name}.png`;
+  let memePath = `/home/sya/garageScript/lesson10/public/memes/${req.body.name}.png`;
+  fs.writeFile(picPath, picData, 'base64', ()=>{
+    gm(picPath).fontSize(40).drawText(50,50,req.body.comment).write(memePath,(err)=>{})
```

Comment:
I don't think it is necessary to have separate paths for meme and picture. Cuz you are not gonna show the picture and it's meaningless to store it in a separate folder.

Reply:
path too long. variable works better

Reply:
overwrote the picPath